.github/gh-actions-self-hosted-runners/arc/environments/beam.env

```diff
@@ -34,9 +34,9 @@ main_runner = {
     runner_image = "us-central1-docker.pkg.dev/apache-beam-testing/beam-github-actions/beam-arc-runner:60d397ecfbd2b10a1929615c70d500eb71a2c053"
     machine_type = "e2-standard-16"
     min_node_count = "1"
-    max_node_count = "16"
+    max_node_count = "24"
     min_replicas = "1"
-    max_replicas = "128"
+    max_replicas = "200"
```

Comment:
Non-blocking - do we have docs on this being related to core count? Eventually we'll want others to be able to manage this, so we should make sure the docs here are obvious

Reply:
Ill add the docs explanation but its basically how requests work in kubernetes. Ill also link appropriate docs to see what is possible